.github/workflows/quarkus.yml

```diff
@@ -51,6 +51,10 @@ on:
   - cron: '0 3 * * *'
   workflow_dispatch:
 
+concurrency:
+  group: ${{github.workflow}} - ${{github.ref}}
```

Comment:
Why `- ${{github.ref}}` here and not in `micronaut.yml`? There are also other workflows that may need a `concurrency` rule such as `spring.yml`.

Reply:
I've added the `${{github.ref}}` because it can also be triggered by pull requests. Therefore, I was assuming that if there are two pr's running this workflow, one should not cancel the other. 
If this is not the case, the `${{github.ref}}` can be removed.

Reply:
In `main.yml`, we've got:
```
group: "workflow = ${{ github.workflow }}, ref = ${{ github.event.ref }}, pr = ${{ github.event.pull_request.id }}"
```

Any reason not to use this consistently? My point was also about being consistent across all other workflows (e.g., micronaut.yml and spring.yml)

Reply:
Ahh I see, sorry for the misunderstanding. 
I've changed it so that it is consistent with the `main.yml`. Same for the `reachabilit-metadata.yml`, `spring.yml` and `micronaut.yml`. 
I did not change the `cdt-inspect.yml` because this is only triggered through a schedule.